packages/react/src/components/Radio/index.tsx

```diff
@@ -11,7 +12,7 @@ export type RadioProps = React.PropsWithChildren<{
 }>
 
 const Radio = forwardRef<HTMLInputElement, RadioProps>(function RadioInner(
-  { value, disabled = false, children, className },
+  { value, disabled = false, children, ...props },
```

Comment:
There is the same issue here.
https://github.com/pixiv/charcoal/pull/523/files#r1602999626

Reply:
fixed